feature-libs/checkout/components/components/payment-method/payment-method.component.html

```diff
@@ -33,6 +33,7 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
               [border]="true"
               [fitToContainer]="true"
               [content]="card.content"
+              [index]="i"
```

Comment:
Should be transferred

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0